src/test/java/com/github/leroyguillaume/keycloak/bcrypt/BCryptPasswordHashProviderTest.java

```diff
@@ -0,0 +1,24 @@
+package com.github.leroyguillaume.keycloak.bcrypt;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.keycloak.models.credential.PasswordCredentialModel;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class BCryptPasswordHashProviderTest {
+    private final int iterations = 10;
+    private final String id = "bcrypt";
+    private final BCryptPasswordHashProvider provider = new BCryptPasswordHashProvider(id, iterations);
+
+    @Test
+    @DisplayName("Should hash the password successfully")
+    void shouldHashThePasswordSuccessfully() {
+        String rawPassword = "test";
+        String hashedPassword = provider.encode(rawPassword, iterations);
+        PasswordCredentialModel model = PasswordCredentialModel.createFromValues(id, new byte[0], iterations, hashedPassword);
+
+        assertNotNull(hashedPassword);
+        assertTrue(provider.verify(rawPassword, model));
+    }
+}
```

Comment:
Can you add an empty line at the end of the file please?

Reply:
Done